fenix/app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -179,7 +179,7 @@ private fun RecentlyVisitedHistoryGroup(
             RecentlyVisitedTitle(
                 text = recentVisit.title,
                 modifier = Modifier
-                    .padding(top = 7.dp, bottom = 2.dp)
+                    .padding(top = 4.dp)
```

Comment:
@Alexandra-Virvara I am just curious how we arrived at these values? Typically, we want to follow the figma specs and be guided by UX. My concern here is that we are changing the spacing drastically which might go again the desired look at that UX is aiming for. That said we should ensure that the text aren't cut off as well.

Reply:
We should also have UX review the visual change and possibly updating the figma spec to follow the same padding values. This is to avoid undoing the changes if anyone had to work with the figma spec on updating anything related to this feature.

Reply:
Some additional thoughts about a different way to go about this. 

@MozillaNoah I believe this is probably caused by the hardcoded size on [L161](https://github.com/mozilla-mobile/firefox-android/pull/1908/files/60760a9fc524995f8e8b0607279e5b333a89e5b1#diff-c1cf213559053edc534f70494d9a5d2d9efad3be191377cddb73eef18f2c9521R161). I think we have already seen this happen elsewhere and we simply removed the hardcoded container size and simply made it "wrap"? Could you confirm?

That way we don't require a UX review while preserving the existing spacing.

Reply:
It's likely caused by the fixed height, but a `Spacer` would need to be added above the `Divider`, otherwise the text may butt right up to the line. 

The text elements, going off of a figma I found, look like they should be using our `TextListItem` Composable, too, but not sure if that's out of the scope for this bug.